autobotAI_integrations/integrations/nessus/__init__.py

```
@@ -0,0 +1,175 @@
+        return {
+            "clients": list_of_unique_elements(cls.get_all_python_sdk_clients()),
+            "supported_executor": "ecs",
+            "compliance_supported": False,
+            "supported_interfaces": cls.supported_connection_interfaces(),
+            "python_code_sample": cls.get_code_sample(),
+            "preview": True,
+        }
+
+    @staticmethod
+    def get_python_sdk_client(
+        connection_details: Union[RestAPICreds, SDKCreds],
```

Ruff (0.15.18)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)